src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -1069,7 +1071,7 @@ static uct_iface_ops_t uct_rc_mlx5_iface_tl_ops = {
     .iface_query              = uct_rc_mlx5_iface_query,
     .iface_get_address        = uct_rc_mlx5_iface_get_address,
     .iface_get_device_address = uct_ib_iface_get_device_address,
-    .iface_is_reachable       = uct_rc_mlx5_iface_is_reachable
+    .iface_is_reachable       = uct_ib_iface_is_reachable
```

Comment:
why? we do not need to change the existing behavior of this func

Reply:
now v2 is the only implementation, it is complete, and now v1 is only wrapping the v2 call, to keep code in one place.